### ISSUE TITLE
Port number '8546' has been specified multiple times. Please review the supplied configuration.

### ISSUE THREAD

**Issue by maxisacoder** [hyperledger/besu-docs#563]
## Describe the bug

- [ ] Missing content
- [ ] Outdated content
- [x] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## The broken page

https://besu.hyperledger.org/en/stable/Tutorials/Permissioning/Create-Permissioned-Network/

## Change suggestion

when start Node-2 using
```
besu --data-path=data --genesis-file=../cliqueGenesis.json --permissions-nodes-config-file-enabled --permissions-accounts-config-file-enabled --rpc-http-enabled --rpc-http-api=ADMIN,ETH,NET,PERM,CLIQUE --host-allowlist="*" --rpc-http-cors-origins="*" --p2p-port=30304 --rpc-http-port=8546
```

will throw error

```
➜  Node-2 besu --data-path=data --genesis-file=../cliqueGenesis.json --permissions-nodes-config-file-enabled --permissions-accounts-config-file-enabled --rpc-http-enabled --rpc-http-api=ADMIN,ETH,NET,PERM,CLIQUE --host-allowlist="*" --rpc-http-cors-origins="*" --p2p-port=30304 --rpc-http-port=8546
2020-12-02 10:30:51,502 main INFO Registered Log4j as the java.util.logging.LogManager.
2020-12-02 10:30:52.579+08:00 | main | INFO  | AbstractAltBnPrecompiledContract | Using LibEthPairings native alt bn128
2020-12-02 10:30:53.286+08:00 | main | INFO  | SECP256K1 | Using native secp256k1
2020-12-02 10:30:53.289+08:00 | main | INFO  | Besu | Starting Besu version: besu/v20.10.1/osx-x86_64/oracle-java-15
Port number '8546' has been specified multiple times. Please review the supplied configuration.

To display full help:
besu [COMMAND] --help
```

the command may need to be changed to 
```
besu --data-path=data --genesis-file=../cliqueGenesis.json --permissions-nodes-config-file-enabled --permissions-accounts-config-file-enabled --rpc-http-enabled --rpc-http-api=ADMIN,ETH,NET,PERM,CLIQUE --host-allowlist="*" --rpc-http-cors-origins="*" --p2p-port=30304 --rpc-http-port=8546 ***--rpc-ws-port=8046(or other)***
```

**Comment by NicolasMassart** [comment]
See fix provided in hyperledger/besu#1665 which is now into Besu master.

**Comment by NicolasMassart** [comment]
[Besu 20.10.3](https://github.com/hyperledger/besu-docs/releases/tag/20.10.3) released today includes the fix for this command line port options
